libs/langgraph/langgraph/pregel/remote.py

```diff
@@ -64,6 +74,15 @@ def __init__(
 
         If `client` or `sync_client` are provided, they will be used instead of the default clients.
         See `LangGraphClient` and `SyncLangGraphClient` for details on the default clients.
+
+        Args:
+            name: The name of the graph.
+            url: The URL of the remote API.
+            api_key: The API key to use for authentication.
```

Comment:
i would add that if not specified, we'll look for `LANGSMITH_API_KEY` etc, similar to the langgraph sdk